force_bdss/core/verifier.py

```diff
@@ -38,13 +36,27 @@ def _check_mco(workflow):
         errors.append(VerifierError(subject=mco,
                                     error="MCO has no defined parameters"))
 
-    for param in mco.parameters:
+    for idx, param in enumerate(mco.parameters):
+        p_name = param.factory.name
         if len(param.name.strip()) == 0:
             errors.append(VerifierError(subject=param,
-                                        error="Parameter has empty name"))
+                                        error="Parameter {} ({}) has empty "
+                                              "name".format(idx, p_name)))
         if len(param.type.strip()) == 0:
             errors.append(VerifierError(subject=param,
-                                        error="Parameter has empty type"))
+                                        error="Parameter {} ({}) has empty "
+                                              "type".format(idx, p_name)))
+
+    for idx, kpi in enumerate(mco.kpis):
+        if len(kpi.name.strip()) == 0:
+            errors.append(VerifierError(subject=kpi,
+                                        error="KPI {} has empty name".format(
+                                            idx)))
+        if kpi.objective == '':
```

Comment:
This technically cannot happen unless we add the empty option to the objective. I am conflicted if we should provide one or not, you made a good point about adding it though...
I'd suggest to add it to the enumeration as well, in first position.